hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml

```diff
@@ -18,4 +18,85 @@
 
 -->
 <server xmlns="urn:jboss:domain:4.0">
+
+  <extensions>
+        <?EXTENSIONS?>
+  </extensions>
+
+  <management>
```

Comment:
I think we’ve used it on JAX-RS Swarm containers that only include JAX-RS, Undertow, and whatever it absolutely requires.

I haven’t noticed it having too many onerous requirements.  But I could also be forgetful.

-Bob


> On Jan 14, 2016, at 4:01 PM, John Mazzitelli <notifications@github.com> wrote:
> 
> In hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml <https://github.com/hawkular/hawkular-agent/pull/137#discussion_r49784371>:
> 
> > @@ -18,4 +18,85 @@
> >  
> >  -->
> >  <server xmlns="urn:jboss:domain:4.0">
> > +
> > +  <extensions>
> > +        <?EXTENSIONS?>
> > +  </extensions>
> > +
> > +  <management>
> I have no attempted to run on Wildfly-Core. I don't even know where that project is at the moment (is that still going to be released now that we have swarm?). But back in the day, the talk was we want hawkular agent to be small enough and not have large amounts of dependencies so it can run inside of WildFly-Core. This was the initial thinking of Heiko Braun when he wrote the first agent prototype - whether it is something we still want to be able to support is something for @theute <https://github.com/theute> and @pilhuhn <https://github.com/pilhuhn> to confirm. I always assumed we would eventually want to install agents inside of WildFly-Core and swarm-based containers that strip out JEE stuff. I wonder if @bobmcwhirter <https://github.com/bobmcwhirter> has tried installing agents in smaller swarm containers?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/hawkular/hawkular-agent/pull/137/files#r49784371>.
>